src/Data/Swagger/Lens.hs

```diff
@@ -62,17 +62,17 @@ makePrisms ''Referenced
 
 _SwaggerItemsArray :: Review (SwaggerItems 'SwaggerKindSchema) [Referenced Schema]
 _SwaggerItemsArray
-  = prism (\x -> SwaggerItemsArray x) $ \x -> case x of
+  = unto (\x -> SwaggerItemsArray x) {- $ \x -> case x of
```

Comment:
Ah, I didn't realise I had `Review` type! I wonder why I abandoned `Prism`... perhaps it did not compile with some old GHC. Thanks anyway!